packages/host/app/components/card-pill.gts

```diff
@@ -49,6 +51,17 @@ export default class CardPill extends Component<CardPillSignature> {
         <div class='card-content' title={{@card.title}}>
           <this.component @format='atom' @displayContainer={{false}} />
         </div>
+        {{#if @onToggle}}
+          <label class={{cn 'toggle' checked=@isEnabled}}>
+            <span class='boxel-sr-only'>Is Enabled:</span>
```

Comment:
I’m not sure what best practice is here but “is enabled” seems maybe redundant as VoiceOver says itself whether the switch is on or off? Maybe a better thing would be to have the name of the card here, as without that it’s unclear what the toggle is actually controlling.

https://github.com/cardstack/boxel/assets/43280/10930338-3bb1-44e0-909a-6b3e49e8d031